About/About.html

```diff
@@ -0,0 +1,135 @@
+    <nav id="nav_bar">
+        <button class="mob_menu" id="mobmenu"><img src="/images/mobile_menu 1.png" alt=""></button>
+        <div id="logo">
+            <a href="/Home/Home.html" class="cool-button">
+                <h1 class="cool-heading">GamerZportal</h1>
+            </a>
+            <div class="nav_elements">
+                <a href="/About/About.html">About</a>
+                <a href="#">News</a>
+                <a href="#">Program</a>
+                <a href="#">Join</a>
+                <a href="#">Sponsor</a>
+            </div>
+        </div>
+    </nav>
+    <!-- main body -->
+    <section class="mainbody">
+        <div class="firstbod">
+            <div class="title">
+                <p id="welcome">
+                    "Welcome to GamerZportal"
+                </p>
+                <p id="desc_2">
+                    Game Forum 2023
+                </p>
+            </div>
+            <div class="description">
+                <p>
+                    Welcome to the Gaming Forum, the ultimate hub for gamers, game developers, and enthusiasts from
+                    around
+                    the
+                    world! Our virtual platform serves as a gathering place where experts, academics, and passionate
+                    gamers
+                    come
+                    together to discuss, explore, and celebrate the fascinating world of gaming.
+                </p>
+            </div>
+            <div class="contact">
+                <div class="part1">
+                    Please contact us per Email for any further questions about Gaming Forum 2023!
+                </div>
+                <div class="email">rdzaman20000@gmail.com</div>
+            </div>
+        </div>
+
+        <section class="part2">
+            <p class="desc5">Gamers Forum
+                Global Summit 2015 Logo</p>
+            <hr>
+            <p class="desc6" id="exc">The logo of CC Global Summit 2015 was decided through the logo competition from 8, June to
+                7, July.</p>
+        </section>
+        <section class="logo">
+            <p id="desc_7">
+                GamerZportal
+            </p>
+        </section>
+        <hr>
+        <section class="past_events">
+            <p class="desc7">See the Past Events</p>
+            <hr>
+            <p class="desc6">Take a look at the past two Gamers Summits which took place in Buenos Aires and in Warsaw
+            </p>
+            <div class="pe">
+                <div class="past_events1">
+                    <p class="date">2012</p>
+                    <p class="place">Gamers summit in warsaw</p>
+                </div>
+                <div class="past_events2">
+                    <p class="date">2015</p>
+                    <p class="place">Gamers summit in Buenos Aeris</p>
+                </div>
+            </div>
+
+        </section>
+        <section class="partners">
+            <p class="desc8">Partners</p>
```

Comment:
- [ ] Please kindly adopt the project design as shown below for the partners section on mobile devices. It is important that your design matches closely the provided template as this tells recruiters and potential clients how much attention to details you pay as a professional frontend developer 👍 
- project design
![pm](https://github.com/Rdzaman20000/GamersPortal/assets/89055502/6c7623c9-4421-48e4-b348-30b43f4affc7)